jmui/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+import os
+
+setup(name='joinmarketui',
+      version='0.9.0dev',
```

Comment:
will need update. The root version string is in `JM_CORE_VERSION` in `jmbase/jmbase/support.py`. I guess we can't avoid multiple hard codings because they're package descriptions.